src/modals/SendModal/index.tsx

```diff
@@ -34,14 +35,28 @@ import ConsolidateUTXOsModal from '../ConsolidateUTXOsModal'
 import SendModalCheckTransaction from './CheckTransaction'
 import SendModalTransactionForm from './TransactionForm'
 
-type StepIndex = 1 | 2 | 3
+enum StepIndex {
+  Send = 1,
+  InfoCheck = 2,
+  PasswordCheck = 3
```

Comment:
Nit: if we change this (more explicit that using numbers 👌 !), why not string literals, which are lighter to use (and less code when compiled)?
```ts
type StepIndex = 'send' | 'infoCheck' | 'passwordCheck'
```

[Related read](https://szaranger.medium.com/whats-an-enum-65db967d8bb6#:~:text=The%20benefits%20of%20using%20a,is%20used%20in%20other%20files.)

Reply:
The only issue with this is the steps can't be iterated vs an enum which constructs both a value and type.

For this context though yea, I think using string literal type is better DevX :slightly_smiling_face: